platform/macos/display_server_macos.mm

```diff
@@ -1897,7 +1897,7 @@
 	bool previously_shown = (mouse_mode == MOUSE_MODE_VISIBLE || mouse_mode == MOUSE_MODE_CONFINED);
 
 	if (show_cursor && !previously_shown) {
-		WindowID window_id = get_window_at_screen_position(mouse_get_position());
+		window_id = get_window_at_screen_position(mouse_get_position());
```

Comment:
Not sure about that one, it's changing behavior. CC @bruvzg

Reply:
window_id is defined above and not used below, but can revert those changes

Reply:
Ah it's probably fine then. I thought it might be a member variable.

Reply:
Seems fine.